src/components/ActionList/ActionList.scss

```diff
@@ -52,15 +52,14 @@ $item-vertical-padding: ($item-min-height - line-height(body)) / 2;
   cursor: pointer;
   padding: $item-vertical-padding spacing(tight);
   border-radius: var(--p-border-radius-base);
-  border-top: 1px solid var(--p-surface); // 1px gap between elements
+  border-top: 1px solid transparent var(--p-surface); // 1px gap between elements
```

Comment:
@ginabak Just an observer – Is `transparent var(--p-surface)` a valid value? Should this be either of them instead?

Reply:
👋🏽 @whizkydee so sorry I missed this comment - just saw this! Does this link answer your question? 😄  https://github.com/Shopify/polaris-react/blob/main/documentation/Color%20system.md#surface Let me know !

Reply:
@ginabak My question is if you're able to apply two colors, not as some sort of gradient to a border – I would imagine that's an invalid value, at least in the current version of CSS. Did you mean to set this to just `transparent`?

Reply:
😅 OOOOH, yes 100%, you're right. I'm going to investigate this. I noticed no visual changes when I remove it! Will ask @alex-page what the reason for this addition was, and 100% I did mean it for to be transparent (only).

Reply:
Haha. Yeah, DevTools probably doesn't yell because of the CSS variable there (maybe the parser isn't smart enough?), but if you use an hex or any other colour format inline, you should get some sort of error indicator.